charts/openzaak/templates/configmap.yaml

```diff
@@ -90,4 +124,12 @@ data:
   {{- end }}
   {{- if .Values.settings.siteDomain }}
   SITE_DOMAIN: {{ .Values.settings.siteDomain | toString | quote }}
+  {{- end }}
+  {{- if .Values.settings.dbDisableServerSideCursors }}
+  DB_DISABLE_SERVER_SIDE_CURSORS: "True"
+  {{- else }}
+  DB_DISABLE_SERVER_SIDE_CURSORS: "False"
+  {{- end }}
```

Comment:
Is there a reason why this is not nested inside `.Values.settings.database` like for all the other env vars that start with `DB_`?